src/components/Game/GameArea.js

```diff
@@ -0,0 +1,138 @@
+import { useState } from 'react';
+
+import ActionButtons from 'components/Game/ActionButtons';
+import History from 'components/Game/History';
+import HealthMeter from "components/Game/HealthMeter";
+
+const PLAYER_ATTACK_MIN_PERCENTAGE = 1;
+const PLAYER_ATTACK_MAX_PERCENTAGE = 10;
+const MONSTER_ATTACK_MIN_PERCENTAGE = 1;
+const MONSTER_ATTACK_MAX_PERCENTAGE = 10;
+const SPECIAL_ATTACK_MIN_PERCENTAGE = 11;
+const SPECIAL_ATTACK_MAX_PERCENTAGE = 20;
+const SPECIAL_ATTACK_MIN_HEALTH = 90;
+const PLAYER_HEAL_PERCENTAGE = 10;
+const GAME_OVER_HEALTH = 0;
+
```

Comment:
move all constants to `game/constants.js`
further, folder/ file names should be in kebab case.